NEWS.md

```diff
@@ -1,5 +1,9 @@
 # camtrapdp (development version)
 
+# camtrapdp 0.2.2
```

Comment:
Don't add this header yet. Any current development goes under `camtrapdp (development version)`